build/build-image/Dockerfile

```diff
@@ -28,5 +28,9 @@ RUN curl -sSL https://github.com/coreos/etcd/releases/download/v3.1.0/etcd-v3.1.
 
 RUN go get -u github.com/golang/lint/golint
 
+RUN git clone https://github.com/duglin/vlinker.git /vlinker
+
+ENV PATH=$PATH:/vlinker/bin
+
```

Comment:
If anything I'd like there to be a comment that says "we're doing the vlinker step" or whatever, through out the file.